### PR TITLE
Modularize mobile timeline wheel

### PR DESCRIPTION
**Pull request by localjo**
Based on `update-build-scripts` (#647) 

Modularizes mobile timeline wheel. Had to revert `mobiscroll` to 2.6.x and put it back in the `/web/ext` directory. The newer version of mobiscroll requires a license to access full functionality, so we need to stick with the old version until we can replace this dependency.

This PR also fixes various mobile bugs.

### CODE REVIEW

**Comment by localjo** [comment]
I'll look into the errors on mobile. Are you seeing the timeline wheel appear on desktop? I wouldn't think it works on desktop.

**Comment by ZachTRice** [comment]
Yeah, the timewheel appears on the responsive desktop view when you click the date in the corner.

**Comment by ZachTRice** [comment]
Note: I am only able to reproduce this when the source is minified.

File: web/js/date/wheels.js [hunk 1]
```diff
@@ -89,3 +89,5 @@ export function dateWheels(models, config) {
   init();
   return self;
 };
+
+export default dateWheels;
```
Comment (Benjaki2) [inline web/js/date/wheels.js]: @localjo what's the reason for making this a default export instead of exporting the `function`?

Comment (localjo) [reply web/js/date/wheels.js]: `dateWheels` is a function. These two are more or less equivalent;

```
export default function(){};
```

And

```
var dateWheels = function(){};
export default dateWheels;
```

I don't remember exactly why I did it this way here. Probably just to be clearer by giving a name to what I was exporting.

**Pull request merged**